subprojects/libero-core/src/main/groovy/com/connexta/libero/Libero.groovy

```diff
@@ -50,34 +52,28 @@ class Libero {
         finalizeConfig(config)
 
         Date date = new Date()
```

Comment:
The main reason is that we sometimes want to use the timestamp that gets generated as part of the release name and git tag name, git does not allow all the characters used by `Instant.now()` in a tag name